txpool/config.go

```diff
@@ -5,8 +5,9 @@ import (
 )
 
 type Config struct {
-	MaxSize int        `toml:"max_size"`
-	Fee     *FeeConfig `toml:"fee"`
+	MaxSize           int        `toml:"max_size"`
+	ConsumptionBlocks uint32     `toml:"-"`
```

Comment:
Let's put the private field after public configs, and add a comment, like: `// Private configs`

Reply:
Is it necessary to have ConsumptionBlocks configured? If the value is constantly equal to 8640 so it would be global constant and not a public field in config